PurchasesCoreSwift/FoundationExtensions/DateFormatter+Extensions.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Created by Andrés Boedo on 7/29/20.
+// Copyright (c) 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+extension DateFormatter {
+    static let iso8601SecondsDateFormatter: DateFormatter = {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ssZ"
+        return dateFormatter
+    }()
+
+    static let iso8601MilliSecondsDateFormatter: DateFormatter = {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZ"
+        return dateFormatter
+    }()
+
+    func iso8601SecondsOrMillisecondsDate(fromBytes bytes: ArraySlice<UInt8>) -> Date? {
```

Comment:
not suuuper fond of this, open to suggestions. 
The part I don't like is that it's an instance method of `dateFormatter`, but it assumes the format. It's name will make that clear, but still.

I think that's why I had it as a separate struct originally.